### DISCUSSION TITLE
How to use ?

### DISCUSSION THREAD

**Discussion by map-sys** [badlydressedboy/azure-data-costs-ui#1] [Q&A]
Hi everyone, just found this and wanted to give it a go but i have no idea how to execute it. If anyone can help ty =)

**Comment by badlydressedboy** [comment]
Hi,

I will put these steps in the readme to help others too. I will also create releases you can download from in the future but for now:

1. Download/install .Net (version 7 or above)
2. Create a local git folder where you want to put the folder containing the code. I use C:⧵git
3. Start a powershell or command prompt then navigate to folder you just created
4. Run: `git clone https://github.com/badlydressedboy/azure-data-costs-ui.git`
5. Change folder to C:\git\azure-data-costs-ui\src\azure-data-costs-ui-wpf
6. Run: `dotnet build .\Azure.Costs.Ui.Wpf.csproj`
7. Run: `dotnet run .\Azure.Costs.Ui.Wpf.csproj`
8. Make sure you have logged into azure through AZ CLI using: `AZ LOGIN` command

**Reply by map-sys** [reply]
Thanks a lot :) will try it
